src/index.html

```diff
@@ -11,7 +11,200 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <!--Block 1 Personal information-->
+      <fieldset class="groups">
+
+        <legend>Personal information</legend>
```

Comment:
```suggestion
      <fieldset class="groups">
        <legend>Personal information</legend>
```
remove blank lines between parent and child elements